dev/travis/before_script.sh

```diff
@@ -76,4 +76,20 @@ case $TEST_SUITE in
 
         cd ../../..
         ;;
+    js)
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
+        nvm install --lts
+        nvm use --lts
+
+        cp package.json.sample package.json
+        cp Gruntfile.js.sample Gruntfile.js
+        npm install -g yarn
```

Comment:
Not only `grunt-cli`, all dependencies defined in `package.json` are downloaded using `yarn` in line 90 (`yarn` is replacement for `npm install`).